proto/ibc/core/channel/v1/tx.proto

```diff
@@ -296,7 +296,11 @@ message MsgChannelUpgradeTry {
 }
 
 // MsgChannelUpgradeTryResponse defines the MsgChannelUpgradeTry response type
-message MsgChannelUpgradeTryResponse {}
+message MsgChannelUpgradeTryResponse {
+  string channel_id       = 1;
+  string version          = 2;
+  uint64 upgrade_sequence = 3;
+}
```

Comment:
should we include some kind of `success` flag to indicate whether or not the TRY handler succeeded successfully or whether an error receipt was written for sequence `n`.